status-im/status-react
# Fixes #3944: Improve metrics opt-in screen (also fixes #3539)

Fixes #3944: Improve metrics opt-in screen
Fixes #3539 "Help improve Status" learn more page

### Summary:

Display enhanced metrics opt-in screen with confirmation yes/no poup per issue 3944.

### Steps to test:
- Open Status
- Follow account creation flow.
- Check appearance of metrics opt-in screen against issue #3944 , and behaviour of yes/no confirmation popup.

status: ready
